test/test_base_service.py

```diff
@@ -398,7 +398,7 @@ def test_request_success_invalid_json():
         service = AnyServiceV1('2018-11-20', authenticator=NoAuthAuthenticator())
         prepped = service.prepare_request('GET', url='')
         service.send(prepped)
-    assert err.value.code == 200
+    assert err.value.status_code == 200
```

Comment:
I think it would be good to have a testcase that still uses the old way `err.value.code` to make sure that still works (see my previous comment about this above)

Reply:
Good idea, I'll add some!